Block/Js.php

```diff
@@ -178,4 +178,13 @@ private function getQuoteIsVirtual() {
         $quote = $this->checkoutSession->getQuote();
         return $quote ? $quote->isVirtual() : false;
     }
+
+    /**
+     * @return string
+     */
+    public function getBoltPopupErrorMessage()
+    {
+        return __('Error') . ': ' . __('Your payment was successful and we\'re now processing your order.' .
```

Comment:
Remove error prefix. We just want to communicate this message. From user's standpoint they don't have to know if it's error (assuming hook can recover the order).